src/anemoi/training/config/training/default.yaml

```diff
@@ -33,11 +33,34 @@ swa:
 # use ZeroRedundancyOptimizer ; saves memory for larger models
 zero_optimizer: False
 
+# loss functions
+
 # dynamic rescaling of the loss gradient
 # see https://arxiv.org/pdf/2306.06079.pdf, section 4.3.2
 # don't enable this by default until it's been tested and proven beneficial
+
+# loss function for the model
+training_loss:
+  # loss class to initialise, can be anything subclassing torch.nn.Module
+  _target_: anemoi.training.losses.mse.WeightedMSELoss
+  # what to include in the loss class initialisation
+  include_node_weights: True
```

Comment:
could these options be briefly described with a comment please?

Reply:
Addressed in 76947331d5306ab6c9fcd7472e4bb068bd525e62